web/styles/app_variables.css

```diff
@@ -1047,6 +1052,15 @@
     --color-background-recent-view-unread-row-hover: hsl(212deg 30% 22% / 60%);
     --color-recent-view-link: hsl(206deg 89% 74%);
     --color-recent-view-link-hover: hsl(208deg 64% 52%);
+    --color-recent-filters-button-focus-background: hsl(
+        0deg 0% 0% / 50%
+    ) !important;
+    --color-recent-filters-button-disabled-background: hsl(0deg 0% 0% / 50%);
+    --color-recent-filters-button-disabled-border: hsl(0deg 0% 0%);
+    --color-recent-view-participant-overflow-text: hsl(0deg 0% 100%) !important;
+    --color-recent-view-participant-overflow-background: hsl(
+        211deg 18% 25%
+    ) !important;
```

Comment:
I don't think `!important` in these variables is a good idea; maybe you can do it in the code below after referencing these, but ideally the CSS selectors should just be explicit enough that we don't need `!important`

Reply:
Fixed.